### PR TITLE
update webauthn section to feature components

### LINKED ISSUE #42595: [BITV]: Note regarding WebAuthn is not accessible
There are contrast issue and screenreader issues (no announce regarding a note).

Note "Your browser does not support WebAuthn." is not accessible:

Screenshot from 2024-01-05 13-23-10

Please create a same Note like in navigation bar settings:

Screenshot from 2024-01-05 12-55-36

### PR DESCRIPTION
**Pull request by szaimen**
Close https://github.com/nextcloud/server/issues/42595

| Before | After |
|---|---|
| image | image |

<details>
<summary>For my own testing</summary>

```
docker run -it --rm ^
--name nextcloud-easy-test ^
-p 8443:443 ^
-e SERVER_BRANCH=enh/42595/webauthn-ncnotecard ^
-e COMPILE_SERVER=1 ^
--volume="nextcloud_easy_test_npm_cache_volume:/var/www/.npm" ^
ghcr.io/szaimen/nextcloud-easy-test:latest

```

</details>

### CODE REVIEW

File: apps/settings/src/components/WebAuthn/AddDevice.vue [hunk 1]
```diff
@@ -25,9 +25,9 @@
 	</div>
 	<div v-else>
 		<div v-if="step === RegistrationSteps.READY">
-			<button @click="start">
+			<NcButton @click="start" type="primary">
```
Comment (ShGKme) [inline apps/settings/src/components/WebAuthn/AddDevice.vue:28]: Doesn't `primay` make too much attention here?

**Review by ShGKme — APPROVED** [review]

File: apps/settings/src/components/WebAuthn/AddDevice.vue (hunk 1 above)
Comment (szaimen) [reply apps/settings/src/components/WebAuthn/AddDevice.vue:28]: I would say primary is fine here. We also use it in the other sections

Comment (susnux) [reply apps/settings/src/components/WebAuthn/AddDevice.vue:28]: Probably would also go with `tertiary` but not I am not a designer.

**Review by susnux — APPROVED** [review]

**Review by emoral435 — APPROVED** [review]

**Comment by szaimen** [comment]
CI failure unrelated IIRC

**Pull request merged**